edb/pgsql/metaschema.py

```diff
@@ -63,6 +63,23 @@ def __init__(self, conn):
         self.db = conn
 
 
+class DBConfigTable(dbops.Table):
+    def __init__(self) -> None:
+        super().__init__(name=('edgedb', '_db_config'))
```

Comment:
I'm just curious: do we need to use a leading `_` for SQL-level tables and helpers?